test/libsolidity/syntaxTests/inlineAssembly/storage_reference_empty_offset.sol

```diff
@@ -1,9 +1,9 @@
 contract C {
     function f() public pure {
         assembly {
-            let x := _offset
+            let x := .offset
         }
     }
 }
 // ----
-// DeclarationError 4794: (84-91): In variable names _slot and _offset can only be used as a suffix.
+// ParserError 1856: (84-85): Literal or identifier expected.
```

Comment:
Why does the parser allows identifiers starting with `.` here?

Reply:
It does not - it says "identifier expected"

Reply:
Right, did not look at the left saying "ParserError".